https://issues.redhat.com/browse/ACM-9851--adding entry to MCE/Observ… (#6201)
open-cluster-management/rhacm-docs: clusters/install_upgrade/acm_integration.adoc

[#acm-integration]
= {product-title-short} integration

> If you are using {mce-short} with {product-title-short} enabled, you can take advantages of further multicluster management features. 

Reviewer: ```suggestion
If you are using {mce-short} with {product-title-short} enabled, you can take advantage of further multicluster management features. 
```

[#integration-prereqs]
== Prerequisites

> See the following prerequisites to integrating with {product-title-short} function. 

Reviewer 2: Shouldn't it be ~~to~~for integrating with?

Author: yes, you are right

> * You need to install {product-title-short}. See link:../../install/install_overview.adoc#[Installing and upgrading] to install.

Reviewer: Will there be more prerequisites listed? If no, i would change the title to be singular

Author: I am meeting with Roke to ask about that later. If not, I will change. There is another issue on this topic.

[#mce-observability]
== Observability integration

> With `multicluster-observability` enabled, you can use {product-title-short} Observability Grafana dashboards to view the following information about your hosted control planes:

Reviewer: ```suggestion
After you enable the `multicluster-observability` pod, you can use {product-title-short} Observability Grafana dashboards to view the following information about your hosted control planes:
```

> - *ACM* > *Hosted Control Planes Overview* dashboard to see cluster capacity estimate for hosting hosted control planes and the related cluster resources.

Reviewer 2: dashboard to see cluster capacity estimate for hosting hosted control planes, the related cluster resources and the list of existing hosted control planes and their status.

Author: changed

> - *ACM* > *Resources* > *Hosted Control Plan* dashboard that you can access from the _Overview_ page to see the resource utilizations of the selected hosted control plane.

Author: ```suggestion
- *ACM* > *Resources* > *Hosted Control Plane* dashboard that you can access from the _Overview_ page to see the resource utilizations of the selected hosted control plane.
```

> To enable, see link:../../observability/observe_environments_intro.adoc#observing-environments-intro[Observability service introduction].

Reviewer 3: plane is misspelled.
- *ACM* > *Resources* > *Hosted Control Plan* dashboard
should be *Hosted Control Plane*
